lib/router/mocks/mediatorMock.js

```diff
@@ -0,0 +1,39 @@
+const q = require('q');
+const sampleUser = require('../fixtures/user');
+
+module.exports = {
+  //eslint-disable-next-line no-unused-vars
+  request: function(topic, args, options) {
```

Comment:
Might want to consider SinonJS here. It's really good for defining function behaviour.